go.mod

```diff
@@ -65,3 +65,5 @@ require (
 	gopkg.in/yaml.v2 v2.2.2
 	gotest.tools v2.2.0+incompatible // indirect
 )
+
+replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```

Comment:
What is this for?

Reply:
This change was required to get `atlantis` to build on my system. I am not a `go` developer and do not know how the `go` module system works, so I cannot give you a definitive explanation. All I can say is that those lines were added by my IDE, apparently because `go.sum` refers to [`git.apache.org/thrift.git`](https://github.com/runatlantis/atlantis/blob/4e8d1e64ac0bc322fc69370607e38cd7600bf4e4/go.sum#L10) which no longer exists, and the content has been moved to https://github.com/apache/thrift

My brief investigation suggested that this was not a direct requirement of `atlantis` but rather a requirement of a module `atlantis` depends on, so I was not able to fix it by directly changing the reference in `go.sum`.  

It is irrelevant to the changes that implement the goals of the PR, but I left it in for the `atlantis` team to see and make a proper fix.